apps/condo/domains/meter/utils/taskSchema/DomaMetersImporter.js

```diff
@@ -64,6 +69,7 @@ class DomaMetersImporter extends AbstractMetersImporter {
                 commissioningDate: row[15],
                 sealingDate: row[16],
                 controlReadingsDate: row[17],
+                isAutomatic: get(this, ['mappers', 'isAutomatic', cell19Value]),
```

Comment:
```suggestion
                isAutomatic: row[19] ? get(this, ['mappers', 'isAutomatic', cell19Value]) : undefined,
```

Reply:
If you have mapping like this: 
```json
{
  "": true
}
```

In case of empty string, you will throw error on line 42, but update meter on line 72. Stay consistent